### PR TITLE
{bio}[foss/2016a] RELION 2.0.1 w/ CUDA (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/r/RELION/RELION-2.0.1-foss-2016a-CUDA-7.5.18.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.0.1'
+commit_id = 'a6225608e270'
```
Comment (boegel) [inline easybuild/easyconfigs/r/RELION/RELION-2.0.1-foss-2016a-CUDA-7.5.18.eb]: same question as in #3683, let's figure it out there and then port whatever we come up with in here

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 38 out of 43 (1 easyconfigs in this PR)
ca040.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f77432b0ac70e5b1a3edeb27e62f608a for a full test report.

**Comment by boegel** [comment]
@verdurin Missing deps in `xprop`:

```
checking for XPROP... configure: error: Package requirements (x11 xproto >= 7.0.17) were not met:
```

Also, missing sources for Mesa?

**Comment by verdurin** [comment]
@boegel - yes, we have an intermittent problem with downloads. Forcing manual `Mesa` download now.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 5 (1 easyconfigs in this PR)
ca040.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/881bed63eb03e87b6bb91a2ad6c1f96a for a full test report.

**Comment by boegel** [comment]
@verdurin That leaves the `xprop` issue...

@hajgato Up for trying to fix that?

**Comment by hajgato** [comment]
@boegel there is a reason why I was not keen to solve that: at `2016a` we do not have "unified" `X11`.

**Comment by boegel** [comment]
@hajgato maybe we can bump this this to `foss/2016b`?

**Comment by hajgato** [comment]
@boegel Yes I can do that, but the we have to go to `CUDA 8` as well

**Comment by boegel** [comment]
@hajgato Well, sure, if RELION works with CUDA 8, why not.

Can you explain why though, I don't see the *need* for CUDA 8 with `foss/2016b`?

**Comment by hajgato** [comment]
@boegel I should put might have to change, I do not know by heart what GCC version supported by CUDA 7.5

**Comment by hajgato** [comment]
@boegel: I have checked, `CUDA 7.5` works up to `GCC 4.9`, so I must update to `CUDA 8` to able to use `GCC 5.4`

**Comment by hajgato** [comment]
@boegel: I have checked, `CUDA 7.5` works up to `GCC 4.9`, so I must update to `CUDA 8` to able to use `GCC 5.4`

**Comment by hajgato** [comment]
This PR is closed. Use #4875 instead. (buped to 2.0.3 w/ 2016b)

**Pull request closed without merging**